sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/FileAPITest.groovy

```diff
@@ -4125,7 +4125,7 @@ class FileAPITest extends APISpec {
 
     @RequiredServiceVersion(clazz = DataLakeServiceVersion.class, min = "V2019_12_12")
     @Retry(count = 5, delay = 5, condition = { environment.testMode == TestMode.LIVE })
-    @PlaybackOnly(expiryTime = "2022-05-18")
+    @PlaybackOnly(expiryTime = "2022-08-18")
```

Comment:
If this date is not updated, will the test fail? Should we let the test fail, so that this gets fixed as this is beyond the expiration date?

Reply:
It does fail if this isn't updated and would block this PR